app/brave_main_delegate.cc

```diff
@@ -30,7 +30,6 @@
 #include "components/autofill/core/common/autofill_payments_features.h"
 #include "components/feed/feed_feature_list.h"
 #include "components/offline_pages/core/offline_page_feature.h"
-#include "components/omnibox/common/omnibox_features.h"
```

Comment:
can we remove any deps for this?

Reply:
Didn't find any deps on //components/omnibox/common